.gitignore

```diff
@@ -8,7 +8,6 @@
 .bower.json
 .sizecache.json
 yarn.lock
-package-lock.json
```

Comment:
You haven't actually added the file to the repo, you've just removed it from `.gitignore`. This is only a part of what's needed here.

Reply:
I have made test in my ubuntu 17.04, I'm not sure if the other distribution will has this issue again.